### DISCUSSION TITLE
Nginx Ingress returns 400 Bad request

### DISCUSSION THREAD

**Discussion by fromano2802** [vitobotta/hetzner-k3s#235] [Q&A]
Hello,

I am following the tutorial at https://github.com/vitobotta/hetzner-k3s/blob/main/wiki/Setting%20up%20a%20cluster.md
and I am able to get to step 17 without any issues, but when I open the hello-world page, it returns a 400 bad request.
I know that the hello-world app is deployed correctly because I can port forward the service and see it in my localhost.

Am I missing anything?
Please advise

**Comment by vitobotta** [comment]
Hi, read the tutorial and the README carefully. If you enable proxy protocol on the load balancer via service annotations, you'll also need to enable it for Nginx via configmap. I'll convert this into a discussion since it's not an issue with the tool.

**Reply by fromano2802** [reply]
Thank you for your reply @vitobotta.
I am not an expert in Kubernetes, if you could help me out with this that would be great.
Also, I read the README, and under the Load Balancers section. you list the options that you use.
 What do you use to configure the LB?

**Reply by fromano2802** [reply]
I figure it out. I added the Nginx configmap and it is working. Thank you

**Reply by vitobotta** [reply]
Glad you got it working :)

**Reply by mabdurrafey1** [reply]
@vitobotta You saved my life, been trying to find the error since two weeks. It was in my destiny to visit this disucssion/issue to resolve the issue. Thanks again

**Reply by fromano2802** [reply]
@mabdurrafey1 I ran into this issue because I didn't read the tutorial entirely.
At step 19 and 20 of the tutorial they indeed explain what to do.
